mix/AssemblyDebuggerControl.cpp

```diff
@@ -198,11 +198,15 @@ void AssemblyDebuggerControl::executeSequence(std::vector<TransactionSettings> c
 
 void AssemblyDebuggerControl::showDebugger(QList<QVariableDefinition*> const& _returnParam, QList<QObject*> const& _wStates, AssemblyDebuggerData const& _code)
 {
+	Q_UNUSED(_reason);
 	m_appEngine->rootContext()->setContextProperty("debugStates", QVariant::fromValue(_wStates));
 	m_appEngine->rootContext()->setContextProperty("humanReadableExecutionCode", QVariant::fromValue(std::get<0>(_code)));
 	m_appEngine->rootContext()->setContextProperty("bytesCodeMapping", QVariant::fromValue(std::get<1>(_code)));
 	m_appEngine->rootContext()->setContextProperty("contractCallReturnParameters", QVariant::fromValue(new QVariableDefinitionList(_returnParam)));
-	this->addContentOn(this);
+	QVariant returnValue;
+	QObject* debugPanel = m_view->findChild<QObject*>("debugPanel", Qt::FindChildrenRecursively);
+	QMetaObject::invokeMethod(debugPanel, "init", Q_RETURN_ARG(QVariant, returnValue));
```

Comment:
onCompleted is fired when the control is loaded on the view.
At this point, the control is already loaded. I will change the name of this function to "update"